dependencies.yaml

```diff
@@ -72,6 +72,16 @@ dependencies:
               cuda: "11.8"
             packages:
               - nvcc_linux-aarch64=11.8
+          - matrix:
+              arch: x86_64
+              cuda: "12.0"
+            packages:
+              - cuda-nvcc_linux-64=12.0
```

Comment:
For other RAPIDS packages I dropped the arch specifier and listed cuda-nvcc with a cuda-version pinning. See cudf.